server/db/migrations/20230309195953_update-campaign-model.js

```diff
@@ -0,0 +1,20 @@
+const tableName = 'campaigns'
+
+module.exports = {
+  async up(knex) {
+    await knex.schema.createTable(tableName, (table) => {
```

Comment:
You are on the right track, but you will need to modify the table and not create a new one with the same name. Look at the [alterTable](https://knexjs.org/guide/schema-builder.html#altertable) method.

Reply:
Thank you! Ok, I switched the function over from createTable to alterTable.